subsys/bluetooth/controller/ll_sw/nordic/lll/lll_clock.c

```diff
@@ -13,26 +13,25 @@
 #define LOG_MODULE_NAME bt_ctlr_llsw_nordic_lll_clock
 #include "common/log.h"
 #include "hal/debug.h"
-
-#define DRV_NAME DT_NORDIC_NRF_CLOCK_0_LABEL "_32K"
-#define K32SRC   CLOCK_CONTROL_NRF_K32SRC
-
-static u8_t is_k32src_stable;
+#include "hal/cpu.h"
 
 void lll_clock_wait(void)
 {
-	if (!is_k32src_stable) {
-		struct device *clk_k32;
+	static bool done;
```

Comment:
initialize with `false` or, better, use int instead of bool. You cannot assume that `false == 0`?

Reply:
As far as I remember, checkpatch complains if you initialize a static to false, as it is always 0

Reply:
Can assume they're equal. The C spec. says they are:

```
They are

    true

which expands to the integer constant 1,

    false

which expands to the integer constant 0
```

`bool` is treated as an unsigned integer type, so it gets zero-initialized.